rllib/utils/policy.py

```diff
@@ -111,14 +111,17 @@ def create_policy_for_framework(
         # and create a new session for it.
         if framework == "tf":
             with tf1.Graph().as_default():
+                # Session creator function provided manually -> Use this one to
+                # create the tf1 session.
                 if session_creator:
                     sess = session_creator()
+                # Use a default session creator, based only on our `tf_session_args` in
+                # the config.
                 else:
                     sess = tf1.Session(
-                        config=tf1.ConfigProto(
-                            gpu_options=tf1.GPUOptions(allow_growth=True)
```

Comment:
don't we need this for fractional gpu?
maybe it doesn't matter since this is tf1 ...

Reply:
We still have this! It's "hidden" in our default values for `config.tf_session_args` (see algorithm_config.py).
And this is where this setting actually should be, not hidden somewhere here in this seemingly random python file :)